### PR TITLE
Formatting + update for Monterey

### PR DESCRIPTION
**Pull request by alyxferrari**
I think I edited every single markdown file in this repository lol. Was in need of some polishing. Updated info for Monterey as well.

### CODE REVIEW

File: types-of-wireless-card/m2.md [hunk 1]
```diff
@@ -46,13 +46,15 @@ To find the PciRoot, run [gfxutil](https://github.com/acidanthera/gfxutil/releas
 
 ![](https://media.discordapp.net/attachments/456913818467958789/681959522432057363/Screen_Shot_2020-02-25_at_1.23.03_PM.png?width=1674&height=895)
 
-All cards presented here **besides the Fenvi and Apple Airport cards** require the following:
+All cards presented here **besides the Fenvi and Apple AirPort cards** require the following:
 
 * [AirportBrcmFixup](https://github.com/acidanthera/AirportBrcmFixup/releases)
 * [BrcmPatchRAM](https://github.com/acidanthera/BrcmPatchRAM/releases)
-  * BrcmBluetoothInjector
+  * BrcmBluetoothInjector (don't use on macOS 12)
```
Comment (1Revenger1) [inline types-of-wireless-card/m2.md:53]: ditto above comment about min/max kernel

**Review by 1Revenger1 — CHANGES_REQUESTED** [review]

File: Kext.md [hunk 2]
```diff
@@ -10,31 +10,34 @@
...
-* Apple Airport and Fenvi cards do not need this kext
+* Note: Apple AirPort and Fenvi cards do not need this kext
 
 ### [BrcmPatchRAM](https://github.com/acidanthera/BrcmPatchRAM/releases)
 
-Required for all non-apple manufactured wireless cards due to how Firmware is handled. This is actually a bundle of multiple kexts:
+Required for all non-Apple manufactured wireless cards due to how firmware is handled. This is actually a bundle of multiple kexts:
 
-* BrcmBluetoothInjector
+* BrcmBluetoothInjector (don't use on macOS 12)
```
[the remark this answers is not in the document]
Comment (alyxxxx) [reply Kext.md]: Resolved with d9dc2b2 and 4348eec

File: Kext.md [hunk 3]
```diff
@@ -10,31 +10,34 @@
...
+  * BrcmPatchRAM for macOS 10.10 and older
+* BlueToolFixup:
+  * Used in place of BrcmBluetoothInjector on macOS 12
 
 Note:
 
-* Apple Airport and Fenvi cards do not need these kexts
-* OpenCore order is alphabetical: Injector -> Data -> RAM
+* Apple AirPort and Fenvi cards do not need these kexts
+* OpenCore order: Injector -> Data -> RAM
```
[the remark this answers is not in the document]
Comment (alyxxxx) [reply Kext.md]: Resolved with 55d2ee5

File: README.md [hunk 4]
```diff
@@ -1,3 +1,3 @@
...
+So you've finally installed macOS onto your desktop/laptop/smart fridge, but you look towards the menubar and see a Bluetooth icon yet no wifi to go with. You immediately bolt to the Settings pane and see no Wi-Fi option and wonder where you may have gone wrong in your install. Scowering the inter-webs did provide an answer, but it was not that which you wanted to hear: "Your Wi-Fi card is SHIT anon!" said some user on a forum with far too many ads and a broken layout. They then proceeded to tell you how stupid you were to even think your wireless card would work. "How dare you not have the knowledge of Hackintosh hive mind!" This page should help clear some things up.
```
[the remark this answers is not in the document]
Comment (alyxxxx) [reply README.md]: Resolved with 3b64130
Didn't know ANON is usually capitalized, thought it was to show intonation 😆

File: types-of-wireless-card/m2.md (hunk 1 above)
Comment (alyxxxx) [reply types-of-wireless-card/m2.md]: Resolved with a42a721

**Comment by alyxxxx** [comment]
I think it should be good to go at this point unless you'd like something else done

**Review by 1Revenger1 — APPROVED** [review]

**Pull request merged**